docs/src/rules/no-return-await.md

```diff
@@ -2,10 +2,12 @@
 title: no-return-await
 rule_type: suggestion
 further_reading:
+- https://v8.dev/blog/fast-async
 - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function
 - https://jakearchibald.com/2017/await-vs-return-vs-return-await/
 ---
 
+This rule was **deprecated** in Eslint v8.49.0 with no replacement. The original intent of this rule no longer applies due to the fact Javascript now handles native `Promises` differently. It can now be slower to remove `await` rather than keeping it. More technical information can be found in [this](https://v8.dev/blog/fast-async) V8 blog entry.
```

Comment:
```suggestion
This rule was **deprecated** in ESLint v8.46.0 with no replacement. The original intent of this rule no longer applies due to the fact Javascript now handles native `Promises` differently. It can now be slower to remove `await` rather than keeping it. More technical information can be found in [this](https://v8.dev/blog/fast-async) V8 blog entry.
```